package.json

```diff
@@ -185,6 +185,7 @@
     "webtreemap-cdt": "^3.2.1"
   },
   "dependencies": {
+    "@puppeteer/replay": "^0.5.0",
```

Comment:
Dep or devDep?

Reply:
https://github.com/GoogleChrome/lighthouse/pull/14146#discussion_r904080943

I'm not 100% sure for this PR, but there are a couple minor reasons we will want this as a dep:
- Makes the stringify extension usable from the node module for anyone who wants it
- Any user flow CLI will need to make this a full dep eventually

I'm also open to leaving it as a dev dep in this PR and circling back when we need to.

Reply:
let's do dev dep for now, until we need it for sure

Reply:
Let's leave as a dev dep.